app/controllers/article_controller.rb

```diff
@@ -131,13 +131,17 @@ def show
       end
     end
 
-    dot_source =
-      render_to_string(:partial => "graph.dot",
-                       :layout => false,
-                       :locals => { :article_links => article_links,
-                                    :link_total => link_total,
-                                    :link_max => link_max,
-                                    :min_rank => min_rank })
+    dot_source = render_to_string(
+      partial: 'graph',
+      layout: false,
+      locals: {
+        article_links: article_links,
+        link_total: link_total,
+        link_max: link_max,
+        min_rank: min_rank
+      },
+      formats: [:dot]
```

Comment:
We added new mime_type :dot, which is handled like erb (technically how it is being handled before, but the updated syntax fails so we needed to add this)